help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,24 +3,29 @@ title: "Configure authentication"
 sidebar_position: 1
 ---
 
-Out-of-the-box, Datagrok offers many authentication methods, including popular methods such as SSO and OAuth.
+Datagrok supports such authentication methods:
+
+* [Login-password](#login-password-authentication)
+* [Oauth](#oauth-authentication)
+* [OpenID](#openid-authentication)
+* [LDAP](#ldap-authentication)
 
 To accommodate any enterprise needs, we can develop a customer-specific integration.
 
-All methods can be enabled separately or combined.
+You can enable all methods separately or combined.
 
 [Learn more about authentification](../govern/authentication.md).
 
 ## Common options
 
-If you want to display something on the login window, set the 'Prompt' option
+To display something on the login window, set the 'Prompt' option
 ![Login prompt](login-prompt.png)
 
 Enable the 'Send Welcome Email' option to send welcome emails to new signed-up users.
 
 ## Login-password authentication
 
-It is the most basic method to authenticate users with Datagrok.
+This is a basic method to authenticate users with Datagrok.
```

Comment:
It is not a good practice to start a paragraph with 'It' or 'This.' And the new sentence is similar to the old one.
Also, It is called General in settings, so it is good to mention it.